htdocs/core/class/html.form.class.php

```diff
@@ -6683,8 +6683,8 @@ function showrefnav($object,$paramid,$morehtml='',$shownav=1,$fieldid='rowid',$f
 
 			// accesskey is for Windows or Linux:  ALT + key for chrome, ALT + SHIFT + KEY for firefox
 			// accesskey is for Mac:               CTRL + key for all browsers
-			$previous_ref = $object->ref_previous?'<a accesskey="p" title="'.$langs->trans("KeyboardShortcut").' ALT+p|ALT+SHIFT+p|CTRL+p" href="'.$navurl.'?'.$paramid.'='.urlencode($object->ref_previous).$moreparam.'"><i class="fa fa-chevron-left"></i></a>':'<span class="inactive"><i class="fa fa-chevron-left opacitymedium"></i></span>';
-			$next_ref     = $object->ref_next?'<a accesskey="n" title="'.$langs->trans("KeyboardShortcut").' ALT+n|ALT+SHIFT+n|CTRL+n" href="'.$navurl.'?'.$paramid.'='.urlencode($object->ref_next).$moreparam.'"><i class="fa fa-chevron-right"></i></a>':'<span class="inactive"><i class="fa fa-chevron-right opacitymedium"></i></span>';
+			$previous_ref = $object->ref_previous?'<a accesskey="p" title="'.$langs->trans("AccessKey").' = p" href="'.$navurl.'?'.$paramid.'='.urlencode($object->ref_previous).$moreparam.'"><i class="fa fa-chevron-left"></i></a>':'<span class="inactive"><i class="fa fa-chevron-left opacitymedium"></i></span>';
```

Comment:
Why no using $conf->browser->os to keep the ALT, or ALT + SHIFT depending on browser...

Reply:
I only changed the title, I wouldn't know where to put `$conf->browser->os`...

or do you mean... `title="'.$conf->browser->os.' = p"'` ?

Reply:
No i mean

$stringforfirstkey = 'CTL'
if ($conf->browser->name == 'chrome') $stringforfirstkey = 'ALT'
if ($conf->browser->name == 'firefox') $stringforfirstkey = 'ALT  SHIFT + KEY'

title="'.$stringforfirstkey.' = p"' ?

So user kown what is the complete shortcut.